<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchEditor</class>
 <widget class="QDialog" name="SearchEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saved Searches</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="Filter">
       <item>
        <property name="text">
         <string>Filter Name:</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Filter All:</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FilterText">
       <property name="toolTip">
        <string>List only the entries containing the text you enter.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="SearchEditorTreeView" name="SearchEditorTree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="Find">
         <property name="toolTip">
          <string>Run Find using the selected entry.
If a group is selected, find the first entry in the list, 
then if not found look for the second entry in the list, etc.</string>
         </property>
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CountAll">
         <property name="toolTip">
          <string>Run Count All for the selected entries in the order selected,
including entries in selected groups.</string>
         </property>
         <property name="text">
          <string>Count All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Replace">
         <property name="toolTip">
          <string>Run Replace using the selected entry.
If a group is selected, replace the first entry in the list,
then when no more matches replace the second entry in the list, etc.</string>
         </property>
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ReplaceAll">
         <property name="toolTip">
          <string>Run Replace All for the selected entries in the order selected,
including entries in selected groups.</string>
         </property>
         <property name="text">
          <string>Replace All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QToolButton" name="MoveUp">
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolTip">
            <string>Move an entry up one level in the same group.</string>
           </property>
           <property name="arrowType">
            <enum>Qt::UpArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="MoveLeft">
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolTip">
            <string>Move an entry to the level of its parent.</string>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="MoveRight">
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolTip">
            <string>You must select an item immediately under a group to move it into the group.</string>
           </property>
           <property name="arrowType">
            <enum>Qt::RightArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="MoveDown">
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolTip">
            <string>Move an entry down one level in the same group.</string>
           </property>
           <property name="arrowType">
            <enum>Qt::DownArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Message">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="toolTip">
        <string>Click Apply to load Find &amp; Replace with the selected search.  Click OK to load your search, save your data, and close.</string>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchEditorTreeView</class>
   <extends>QTreeView</extends>
   <header>MiscEditors/SearchEditorTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
